libs/vectorscan/Makefile

```diff
@@ -0,0 +1,79 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vectorscan
+PKG_VERSION:=5.4.11
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/VectorCamp/vectorscan/tar.gz/$(PKG_NAME)/$(PKG_VERSION)?
+PKG_HASH:=905f76ad1fa9e4ae0eb28232cac98afdb96c479666202c5a4c27871fb30a2711
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=John Audia <therealgraysky@proton.me>
+PKG_LICENSE:=BSD-3-Clause BSD-2-Clause BSL-1.0
+PKG_LICENSE_FILES:=LICENSE
+CMAKE_INSTALL:=1
+
+PKG_BUILD_DEPENDS:=ragel/host python3/host boost/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_OPTIONS += \
+	-DCMAKE_INSTALL_PREFIX=/usr \
+	-DCMAKE_INSTALL_LIBDIR=lib \
+	-DBUILD_SHARED_LIBS=ON \
+	-Wno-dev
+
+define Package/vectorscan-headers
+  CATEGORY:=Libraries
+  SECTION:=libs
+  TITLE:=Vectorscan Headers
+  URL:=https://github.com/VectorCamp/vectorscan
+  DEPENDS:=@aarch64
```

Comment:
Hm... not sure if we hadnt have this discussion elsewhere, but according to readme in Vectorscan, I see that it is fork from hyperscan, but with more platforms supported. So in theory, x86 should be supported and there should be added conflict as it was pointed out by someone else in this PR.

Reply:
I added the CONFLICTS.  Please let me know if syntax and location are correct.

Reply:
You should not mark it as resolved,  it should be moved somewhere else. I will help you on this, but not tonight.

Reply:
Unmarked, thanks for the help.  I studied a few other Makefiles and see that it belongs in the Package define statement.  Moved.  Let me know if it is OK as-is or needs to be `CONFLICTS:=hyperscan-runtime` instead of `=hyperscan` or something else.

Reply:
@BKPepe - is this change correct?

Reply:
@BKPepe - friendly ping to review